libs/ui/elements/src/lib/markdown-parser/markdown-parser.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core'
+import { marked } from 'marked'
+
+@Component({
+  selector: 'gn-ui-markdown-parser',
+  templateUrl: './markdown-parser.component.html',
+  styleUrls: ['./markdown-parser.component.css'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class MarkdownParserComponent {
```

Comment:
We usually try to add presentation components to storybook, do you think you could do the same with this one?
I suggest you take inspiration from other components such as this one: https://github.com/geonetwork/geonetwork-ui/blob/main/libs/ui/elements/src/lib/metadata-info/linkify.directive.stories.ts

The goal is to let people easily "try" the component and see its behaviour with varying inputs, and also making sure that it doesn't break after modifications.